Sources/Fluent/Query/Comparison.swift

```diff
@@ -28,8 +28,20 @@ public func ==(_ lhs: Filter.Comparison, _ rhs: Filter.Comparison) -> Bool
     switch (lhs, rhs) {
     case (.like(let leftPos), .like(let rightPos)):
         return leftPos == rightPos
+    case (.equals, .equals):
```

Comment:
I'm pretty sure you have to implement the entire enum in equatable. That's not a huge issue, though, because the compiler will require all cases to be there if you don't have a `default`